OutOfSchool/OutOfSchool.DataAccess/Enums/DaysOfWeek.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace OutOfSchool.Services.Enums
+{
+    public enum DaysOfWeek
```

Comment:
DayOfWeek enum is available in the framework
https://docs.microsoft.com/en-us/dotnet/api/system.dayofweek?view=net-5.0